keps/sig-node/2238-liveness-probe-grace-period/README.md

```diff
@@ -168,6 +177,12 @@ down upon failure.
 
 ### Test Plan
 
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
```

Comment:
are there any suggested new tests/test updates, as of now for v1.25 for enabling with the default on?

Reply:
Yes, will add them to the doc